packages/tokens/scripts/generate-css-var-map.js

```diff
@@ -0,0 +1,119 @@
+const { argv } = require("node:process"); // For accessing command line arguments
+const { format, normalize } = require("node:path"); // For path manipulation
+const path = require("node:path"); // For path operations
+const { writeFile, mkdir } = require("node:fs/promises"); // For file system operations (async)
+
+// Constant for the prefix used in CSS variables
+const TELEGRAPH_VARIABLE_PREFIX = "tgph";
+
+/**
+ * Maps CSS variables to class names based on the provided object structure.
+ * @param {Object} obj - The object containing design tokens.
+ * @param {String} prefix - Prefix for the class name.
+ * @param {Boolean} noVariableNamePrefix - Flag to include/exclude the prefix in variable names.
```

Comment:
Tiny nit: I think it might be better to flip this to have it as "includeVariableNamePrefix" where this defaults to true. Then you don't end up doing the double not `!no` in the one place we use it.